From 20b48568070483091f5157fc3456c6f10870a3c3 Mon Sep 17 00:00:00 2001
From: Peter W Morreale <pmorreal@brocade.com>
Date: Fri, 21 Feb 2014 10:54:08 -0700
Subject: [PATCH 4/5] update man page.

Update the man page to document the new 'local ip range' option.

Signed-off-by: Peter W Morreale <pmorreal@brocade.com>
---
 doc/xl2tpd.conf.5 | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/doc/xl2tpd.conf.5 b/doc/xl2tpd.conf.5
index 74c58bc..3df8561 100644
--- a/doc/xl2tpd.conf.5
+++ b/doc/xl2tpd.conf.5
@@ -120,6 +120,19 @@ more information about hidden AVP's and AVP in general, refer to rfc2661
 .B local ip
 Use the following IP as xl2tpd's own ip address.
 
+.TP
+.B local ip range
+Specify the range of addresses the LNS will assign as the local address
+to connecting LAC PPP tunnels.  This option is mutually exclusive with
+the
+.B local ip
+option and is useful in cases where it is desirable to have a unique IP
+address for each tunnel.  Specify the range value exactly like the
+.B ip range
+option.  Note that the
+.B assign ip
+option has no effect on this option.
+
 .TP 
 .B length bit
 If set to yes, the length bit present in the l2tp packet payload
-- 
1.8.4.5

